config/crd/bases/apps.kruise.io_advancedcronjobs.yaml

```diff
@@ -181,6 +181,7 @@ spec:
                   x-kubernetes-preserve-unknown-fields: true
               type: object
           required:
+          - concurrencyPolicy
```

Comment:
Do you need to remove this as well?

Reply:
Generated manifest again in commit https://github.com/openkruise/kruise/pull/468/commits/038cb120e2e87f5fed7c82e5b8e266d66a02bb58